doc/rst/source/batch.rst

```diff
@@ -0,0 +1,273 @@
+    path is given then we create a working directory in the system temp folder named *prefix*.  The main benefit
+    of a working directory is to avoid endless syncing by agents like DropBox or TimeMachine, or to avoid
+    problems related to low space in the main directory.  The product files are still placed in the *prefix* directory.
+    The *workdir* is removed unless **-Q** is used for debugging.
+
+.. _-Z:
+
+**-Z**
+    Erase the *mainscript* and all input scripts given via **-I** and **-S** upon completion (except when **-Q** is used).
+
+.. _-cores:
+
+**-x**\ [[-]\ *n*]
+    Limit the number of cores used when making the individual jobs.
+    By default we try to use all available cores.  Append *n* to only use *n* cores
+    (if too large it will be truncated to the maximum cores available).  Finally,
+    give a negative *n* to select (all - *n*) cores (or at least 1 if *n* equals or exceeds all).
+    The parallel processing does not depend on OpenMP.
+
+.. include:: explain_help.rst_
+
+Parameters
+----------
+
+Several parameters are automatically assigned and can be used when composing *mainscript* and the optional
+*preflight* and *postflight* scripts. There are two sets of parameters: Those that are constants
+and those that change with the job number.  The constants are accessible by all the scripts:
+**BATCH_PREFIX**\ : The common prefix of the batch jobs (set with **-N**).
+**BATCH_NJOBS**\ : The total number of jobs.
+Also, if **-I** was used then any static parameters listed there will be available to all the scripts as well.
+In addition, the *mainscript* also has access to parameters that vary with the job counter:
+**BATCH_JOB**\ : The current job number (an integer, e.g., 136),
+**BATCH_TAG**\ : The formatted job number (a string, e.g., 000136), and
+**BATCH_NAME**\ : The name prefix for the current job (i.e., *prefix*\ _\ **BATCH_TAG**),
+Furthermore, if a *timefile* was given then variables **BATCH_COL0**\ , **BATCH_COL1**\ , etc. are
+also set, yielding one variable per column in *timefile*.  If *timefile* has trailing text then that text can
+be accessed via the variable **BATCH_TEXT**, and if word-splitting was explicitly requested by **-T+w** then
+the trailing text is also split into individual word parameters **BATCH_WORD0**\ , **BATCH_WORD1**\ , etc.
+**Note**: Any product(s) made by the processing scripts should be named using **BATCH_NAME** as their prefix
+as these will be automatically moved up to the starting directory upon completion.
+
+Data Files
+----------
+
+The batch scripts will be able to find any files present in the starting directory when **batch** was initiated,
+as well as any new files produced by *mainscript* or the optional scripts set via **-S**.
+No path specification is needed to access these files.  Other files may
+require full paths unless their directories were already included in the :term:`DIR_DATA` setting.
+
+Plotting Temporal Changes
+-------------------------
```

Comment:
Plotting? Should be changed to other words.

Reply:
Fixed. Cut/paste from movie.